Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY: install test run_no_staking_tests
+
+# Install project dependencies
+install:
+	poetry install
```

Comment:
`poetry install` is done in every `.sh` file like `run_service.sh`, `stop_service.sh`, etc. So we'll need to update those also like `poetry install --only main` in them, so that the user doesn't bloat their space with dev dependencies.

Reply:
sure, updated all *.sh files